src/api/libcellml/logger.h

```diff
@@ -0,0 +1,121 @@
+/*
+Copyright 2015 University of Auckland
```

Comment:
We might need to do a library-wide search and replace for this after this pull request. And maybe also update CellML 1.2 to 2.0 in the same tidy-up pull.